django_x509/static/django-x509/css/extensions-widget.css

```
@@ -0,0 +1,125 @@
+  border: 1px solid var(--border-color);
+  border-radius: 6px;
+  color: var(--body-fg);
+  margin-bottom: 0.75rem;
+  padding: 0.9rem;
+}
+
+.x509-extensions-row__header {
+  align-items: start;
+  column-gap: 1rem;
+  display: grid;
+  gap: 0.75rem;
```

Biome (2.4.6)

[error] Unexpected shorthand property gap after column-gap

(lint/suspicious/noShorthandPropertyOverrides)